### DISCUSSION TITLE
Unable to execute curl commands inside openpolicyagent/opa:0.64.1-istio

### DISCUSSION THREAD

**Discussion by bvamshidhar** [open-policy-agent/community#614] [Community]
@ashutosh-narkar 
This is the error I get when I try to run curl inside the opa sidecar container:

"command": [
        "curl",
        "--output-dir",
        "/test-dir",
        "-s",
        "-O",
        "https://example.url.com"
      ],

Reason:       StartError
      Message:      failed to create containerd task: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: exec: "curl": executable file not found in $PATH: unknown
      

Can you help with identifying the base image of this sidecar container?
How can install curl onto this. Can you give the snippet if possible. 

P.S: I'm using [openpolicyagent/opa:0.64.1-istio](https://hub.docker.com/layers/openpolicyagent/opa/0.64.0-istio/images/sha256-cce6f76afd7e2d0050f321c733e982643cd0538ce92c4f2f018af25c84901bc2?context=explore) image

**Comment by ashutosh-narkar** [comment]
You can see the base images [here](https://github.com/open-policy-agent/opa-envoy-plugin/blob/main/Makefile#L94-L102).  They do not come with a shell. You could use something like `chainguard/glibc-dynamic:latest-dev` to build a custom image that will have a shell.
